setup.py

```diff
@@ -235,6 +235,7 @@ def copy_data_tree(datadir, destdir):
         [rasterio.rio_commands]
         bounds=rasterio.rio.features:bounds
         calc=rasterio.rio.calc:calc
+        clip=rasterio.rio.features:clip
```

Comment:
I feel like clip could go in rasterio.rio.convert since rio-convert is so closely related: they are two commands that are joined in gdal_translate.